scripts/runner-env.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
```

Comment:
can we call the file `runner-update-env.js`?

Reply:
Indeed

Reply:
Fixed in 8338a3b6